### PR TITLE
MM-28306 Cypress/E2E: For MM-T670, MM-T672 and MM-T674

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary

Jira ticket: https://mattermost.atlassian.net/browse/MM-28306

Add Cypress tests for:
- [MM-T670](https://automation-test-cases.vercel.app/test/MM-T670)
- [MM-T672](https://automation-test-cases.vercel.app/test/MM-T672)
- [MM-T674](https://automation-test-cases.vercel.app/test/MM-T674)

### CODE REVIEW

**Review by reflog — APPROVED** [review]

**Review by larkox — APPROVED** [review]
LGTM

**Comment by lindy65** [comment]
Hi @saturninoabril - can you help link the Jira ticket for these Cypress tests please?

**Comment by saturninoabril** [comment]
No Jira ticket but I created one now. See updated title and description.  Thanks Lindy.

File: e2e/cypress/integration/integrations/builtin_commands/user_status_spec.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+        verifyUserStatus(online);
+    });
+});
+
+function verifyUserStatus(testCase) {
+    // # Post slash command to change user status
+    cy.postMessage(testCase.command);
+
+    // * Get last post and verify system message
```
Comment (lindy65) [inline e2e/cypress/integration/integrations/builtin_commands/user_status_spec.js:61]: Hi @saturninoabril - in test case MM-T672 (/offline) there are two verification steps:
- to switch to another channel and back to the channel where the system message posted saying you're offline so you can verify `No "New Messages" line appears above system message "You are now offline"`
- `System message should not display online indicator on the profile picture (this is true for all system messages)`
Is it possible to include these verifications in the PR?

Comment (saturninoabril) [reply e2e/cypress/integration/integrations/builtin_commands/user_status_spec.js:81]: Oh thanks Lindy for catching. It's the reason why I didn't include `/dnd` but failed to apply :).  Should be good now. Also, I modified the test case and combined the step into a single script.

Comment (lindy65) [reply e2e/cypress/integration/integrations/builtin_commands/user_status_spec.js:81]: Perfect, thanks Saturn 👍 I'll re-review :)

**Review by lindy65 — APPROVED** [review]
Thanks so much @saturninoabril - looks good to me 👍

**Pull request merged**